api/api.js

```diff
@@ -14,7 +15,13 @@ api.post("/subscribe", async (req, res) => {
 			res.status(400).json({ sucess: false, message: "bad request" });
 		}
 
-		res.status(200).json({ sucess: true });
+		const user = await lookupEmail(email);
+
+		if (user.ok) {
+			// Insert users data into DB here...
```

Comment:
A nice trick with to do statements is to use the format
```suggestion
			// @todo Insert users data into DB here
```

This is using [JSDoc syntax](https://jsdoc.app/tags-todo) which most IDEs and other code tooling recognise so it gives you lots of benefits such as syntax highlighting and linting rules